.travis.yml

```diff
@@ -0,0 +1,25 @@
+language: python
+python: 3.5
+
+env:
+  - TOXENV=py35
+  - TOXENV=flake8
+
+before_install:
+  - source /etc/lsb-release
+  - echo "deb http://download.rethinkdb.com/apt $DISTRIB_CODENAME main" | sudo tee -a /etc/apt/sources.list.d/rethinkdb.list
+  - wget -qO- https://download.rethinkdb.com/apt/pubkey.gpg | sudo apt-key add -
+  - sudo apt-get update -qq
+
+install:
+  - sudo apt-get install rethinkdb
+  - pip install "git+https://github.com/bigchaindb/bigchaindb.git"
```

Comment:
Once we release the new version of BigchainDB, this should be changed to just be `pip install bigchaindb`